### PR TITLE
please confirm translation

### PR DESCRIPTION
**Pull request by drelaptop**
translation waiting for confirm = "[精灵表单 Using Spritesheets]()"
waiting for translation =   [Queries 如何翻译]()

### CODE REVIEW

File: zh/SUMMARY.md [hunk 1]
```diff
@@ -3,113 +3,113 @@
...
+    - [图形性能调优 Optimizing Graphics]()
+    - [SQLite集成]()
 
 - [VR(虚拟现实)](README.md)
-    - [快速上手](翻译是否合适?是否考虑将本章放入[Advanced-Topics])
+    - [快速上手]()
 
 - [环境搭建](./installation/index.md)
-    - [系统要求](installation/A.md)
+    - [安装要求](installation/A.md)
```
Comment (drelaptop) [inline zh/SUMMARY.md]: 类似 Android 和 Visual Studio 的英文,不翻译?

File: zh/SUMMARY.md [hunk 2]
```diff
@@ -3,113 +3,113 @@
...
+    - [关于我们]()
     - [引擎优势]()
-    - [How to learn](warn内容少,建议融入[获取帮助],去除本节)
+    - [如何学习]()
     - [获取帮助]()
     - [支持]()
 
 - [基本概念](README.md)
-    - [Main Components](info建议融到本章介绍,去除本节)
+    - [主要组件 Main Components]()
```
Comment (drelaptop) [inline zh/SUMMARY.md]: please confirm or modify

File: zh/SUMMARY.md [hunk 3]
```diff
@@ -3,113 +3,113 @@
...
+    - [精灵控制]()
+    - [多边形精灵 Polygon Sprite]()
 
 - [动作](README.md)
-    - [简介]()
+    - [动作概念]()
     - [基本动作](./action/zh.md)
     - [动作序列]()
-    - [序列原理](info翻译是否合适)
+    - [序列原理 Sequence Internals, what is really happening]()
```
Comment (drelaptop) [inline zh/SUMMARY.md]: please confirm or modify, 4 item up

File: zh/SUMMARY.md [hunk 4]
```diff
@@ -3,113 +3,113 @@
...
 - [使用脚本](README.md)
     - [脚本组件]()
 
 - [物理引擎](README.md)
-    - [Physics is scary, do I really need it? Please tell me no!](warn介绍性质,建议放于本章介绍,在中文文档中去除本节)
+    - [物理引擎简介]()
     - [术语与概念]()
     - [碰撞]()
-    - [Queries](error本节如何翻译?)
+    - [Queries 如何翻译]()
```
Comment (drelaptop) [inline zh/SUMMARY.md]: how to translate

File: zh/SUMMARY.md [hunk 5]
```diff
@@ -118,11 +118,12 @@
...
         - [Visual Studio](installation/Windows.md)
+        - [Windows Phone](installation/Windows-Phone.md) __**DEPRECATED**__
 
 - [工具](README.md)
     - [Cocos 命令行工具]()
-    - [Cocos 图形界面]()
-    - [构建运行库](warn翻译是否合适)
\ No newline at end of file
+    - [Cocos 图形界面 Cocos GUI]()
+    - [构建运行库 Pre-built libraries]()
```
Comment (drelaptop) [inline zh/SUMMARY.md]: all Items like "[中文 English]()" need to be confirm

File: zh/SUMMARY.md (hunk 1 above)
Comment (minggo) [reply zh/SUMMARY.md]: 需要翻译，可以参考别的文档是怎么写的。

File: zh/SUMMARY.md (hunk 2 above)
Comment (minggo) [reply zh/SUMMARY.md]: 主要组件 Main Components -> 主要组件(Main Components)

File: zh/SUMMARY.md (hunk 5 above)
Comment (minggo) [reply zh/SUMMARY.md]: should change to 中文(English)

File: zh/SUMMARY.md (hunk 4 above)
Comment (minggo) [reply zh/SUMMARY.md]: it is a concept of physics engine? i have not idea with it, you can refer to physics engine docs to check its meaning.

File: zh/SUMMARY.md (hunk 3 above)
Comment (minggo) [reply zh/SUMMARY.md]: 序列原理 -> 序列动作原理
others looks good to me

File: zh/SUMMARY.md (hunk 5 above)
Comment (drelaptop) [reply zh/SUMMARY.md]: understood

File: zh/SUMMARY.md [hunk 6]
```diff
@@ -3,126 +3,129 @@
...
-    - [Linux](installation/Linux.md)
-    - [Tizen]()
-    - [Windows](installation/Windows.md)
+    - [IOS平台](installation/iOS.md)
+    - [macOS平台](installation/OSX.md)
+    - [Linux平台](installation/Linux.md)
+    - [Tizen平台]() __**DEPRECATED**__
+    - [Windows平台](installation/Windows.md)
         - [Visual Studio](installation/Windows.md)
+        - [Windows Phone](installation/Windows-Phone.md) __**DEPRECATED**__
```
Comment (drelaptop) [inline zh/SUMMARY.md]: Deprecated `[Tizen平台]()` and `[Windows Phone]()` need to be removed or not?

Comment (minggo) [reply zh/SUMMARY.md]: remove them

File: zh/SUMMARY.md [hunk 7]
```diff
@@ -3,126 +3,129 @@
...
         - [Eclipse](installation/Android-Eclipse.md)
-    - [IOS](installation/iOS.md)
-    - [Mac OS X](installation/OSX.md)
-    - [Linux](installation/Linux.md)
-    - [Tizen]()
-    - [Windows](installation/Windows.md)
+    - [IOS平台](installation/iOS.md)
+    - [macOS平台](installation/OSX.md)
+    - [Linux平台](installation/Linux.md)
+    - [Tizen平台]() __**DEPRECATED**__
```
Comment (minggo) [inline zh/SUMMARY.md]: not needed, can remove it

File: zh/SUMMARY.md [hunk 8]
```diff
@@ -3,126 +3,129 @@
...
         - [Visual Studio](installation/Windows.md)
+        - [Windows Phone](installation/Windows-Phone.md) __**DEPRECATED**__
 
 - [工具](README.md)
     - [Cocos 命令行工具]()
-    - [Cocos 图形界面]()
-    - [构建运行库](warn翻译是否合适)
\ No newline at end of file
+    - [Cocos 图形界面工具]()
+    - [提前构建运行库]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 这个是 预编译库 吧

File: zh/SUMMARY.md [hunk 9]
```diff
@@ -3,126 +3,129 @@
...
         - [命令行](installation/Android-terminal.md)
         - [Visual Studio](installation/Android-VisualStudio.md)
         - [Android Studio](./installation/android-studio-intallation/installation.md)
         - [Eclipse](installation/Android-Eclipse.md)
-    - [IOS](installation/iOS.md)
-    - [Mac OS X](installation/OSX.md)
-    - [Linux](installation/Linux.md)
-    - [Tizen]()
-    - [Windows](installation/Windows.md)
+    - [IOS平台](installation/iOS.md)
```
Comment (minggo) [inline zh/SUMMARY.md]: iOS

File: zh/SUMMARY.md [hunk 10]
```diff
@@ -3,126 +3,129 @@
...
+    - [SimpleAudioEngine与AudioEngine]()
 
 - [高级话题](README.md)
-    - [本章标题是否合适?]()
     - [文件接入]()
     - [网络操作]()
     - [材质与纹理]()
-    - [图形性能调优](warn翻译是否合适?)
-    - [SQLite]()
+    - [图形性能调优]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 图形性能调优 -> 图形性能优化

File: zh/SUMMARY.md [hunk 11]
```diff
@@ -3,126 +3,129 @@
...
+    - [SQLite集成]()
+
+- [虚拟现实(VR)](README.md)
+    - [快速了解]()
```
Comment (minggo) [inline zh/SUMMARY.md]: can remove it

File: zh/SUMMARY.md [hunk 12]
```diff
@@ -3,126 +3,129 @@
...
 
-- [声音](README.md)
-    - [Getting Started](warn内容少,介绍性质,建议放到本章介绍,去除本节)
+- [声音和音效](README.md)
+    - [简介]()
     - [背景音乐](./audio-and-effect/zh.md)
     - [声音控制]()
-    - [高级功能](warn翻译是否合适?)
+    - [高级功能]()
+    - [SimpleAudioEngine与AudioEngine]()
```
Comment (minggo) [inline zh/SUMMARY.md]: SimpleAudioEngine与AudioEngine -> SimpleAudioEngine与AudioEngine对比

File: zh/SUMMARY.md (hunk 8 above)
Comment (drelaptop) [reply zh/SUMMARY.md]: 英文是 `Pre-built libraries`, 内容是空的, 我改一下

File: zh/SUMMARY.md [hunk 13]
```diff
@@ -3,126 +3,129 @@
...
-    - [序列]()
-    - [Parent Child Relationship](warn和并列的节,不属于一类,建议融到入[节点]和[精灵]的内容中)
-    - [日志]()
+    - [组件简介]()
+    - [导演(Director)]()
+    - [场景(Scene)]()
+    - [精灵(Sprite)]()
+    - [动作(Action)]()
+    - [序列(Sequence)]()
+    - [组件关系]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 组件关系 -> 组件结构

File: zh/SUMMARY.md [hunk 14]
```diff
@@ -3,126 +3,129 @@
 ## 新手入门
 
 - [了解引擎](README.md)
-    - [关于]()
+    - [关于](README.md)
     - [引擎优势]()
-    - [How to learn](warn内容少,建议融入[获取帮助],去除本节)
+    - [如何学习]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 如何学习 -> 学习资源

File: zh/SUMMARY.md [hunk 15]
```diff
@@ -3,126 +3,129 @@
...
-    - [光照]()
-    - [地表](warn翻译是否合适)
-    - [3D工具]()
-    - [高阶话题](warn翻译是否合适)
+    - [术语]()
+    - [3D精灵(Sprite3D)]()
+    - [3D动画(Animation3D)]()
+    - [相机(Camera)]()
+    - [方块纹理(TextureCube)]()
+    - [模拟天空(Skybox)]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 模拟天空 -> 天空盒

File: zh/SUMMARY.md [hunk 16]
```diff
@@ -3,126 +3,129 @@
...
-    - [模拟天空](warn翻译是否合适)
-    - [光照]()
-    - [地表](warn翻译是否合适)
-    - [3D工具]()
-    - [高阶话题](warn翻译是否合适)
+    - [术语]()
+    - [3D精灵(Sprite3D)]()
+    - [3D动画(Animation3D)]()
+    - [相机(Camera)]()
+    - [方块纹理(TextureCube)]()
```
Comment (minggo) [inline zh/SUMMARY.md]: 方块纹理 -> 立方体纹理

File: zh/SUMMARY.md (hunk 14 above)
Comment (drelaptop) [reply zh/SUMMARY.md]: I will modify them, and update PR

**Comment by drelaptop** [comment]
have modified, please review

**Pull request merged**